.github/workflows/lint.yml

```diff
@@ -62,4 +62,20 @@ jobs:
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@2.0.0
         with:
-          scandir: './libevm'
+          scandir: "./libevm"
+
+  markdown:
```

Comment:
Add `markdown` to the `needs` of the primary `lint` job otherwise this will never be enforced.